Prox/Prox/Data/EventsController.swift

```diff
@@ -15,15 +15,15 @@ class EventsProvider {
     lazy var eventsDatabase: EventsDatabase = FirebaseEventsDatabase()
 
     private lazy var radius: Double = {
-        return RemoteConfigKeys.getDouble(forKey: RemoteConfigKeys.eventSearchRadiusInKm)
+        return RemoteConfigKeys.eventSearchRadiusInKm.value
     }()
 
     private lazy var eventStartNotificationInterval: TimeInterval = {
-        return RemoteConfigKeys.getTimeInterval(forKey: RemoteConfigKeys.eventStartNotificationInterval)
+        return RemoteConfigKeys.eventStartNotificationInterval.value * 60
```

Comment:
I considered this, but separating how the constant is stored, and how its used seemed to be a useful distinction here. 

Suggest an `TimeInterval` `extension` to manage converting to seconds from minutes. e.g. 

```swift
extension TimeInterval {
    init(minutes: Double) {
        self.init(minutes * 60)
    }

    init(hours: Double) {
        self.init(minutes: hours * 60)
    }
}

```